src/Mod/Unit.h

```diff
@@ -63,6 +82,45 @@ struct UnitStats
 	UnitStats operator-() const { return UnitStats(-tu, -stamina, -health, -bravery, -reactions, -firing, -throwing, -strength, -psiStrength, -psiSkill, -melee, -mana); }
 	void merge(const UnitStats& stats) { tu = (stats.tu ? stats.tu : tu); stamina = (stats.stamina ? stats.stamina : stamina); health = (stats.health ? stats.health : health); bravery = (stats.bravery ? stats.bravery : bravery); reactions = (stats.reactions ? stats.reactions : reactions); firing = (stats.firing ? stats.firing : firing); throwing = (stats.throwing ? stats.throwing : throwing); strength = (stats.strength ? stats.strength : strength); psiStrength = (stats.psiStrength ? stats.psiStrength : psiStrength); psiSkill = (stats.psiSkill ? stats.psiSkill : psiSkill); melee = (stats.melee ? stats.melee : melee); mana = (stats.mana ? stats.mana : mana); };
 
+	/**
+	 * @brief Gets the attribute identified by a text string. 
+	 * @param attribute the attribute to get. 
+	 * @return a const reference to the attribute.
+	*/
+	const Type &operator[](StatType attribute) const
```

Comment:
This function and `enum` can be removed, use directly `&UnitStats::tu` in `absoluteConditionData`

Reply:
Great idea, I should have thought of that!